newIDE/app/src/Profile/AuthenticatedUserProvider.js

```diff
@@ -75,20 +78,22 @@ export default class AuthenticatedUserProvider extends React.Component<
         onEdit: () => this.openEditProfileDialog(true),
         onCreateAccount: () => this.openCreateAccountDialog(true),
         onRefreshUserProfile: this._fetchUserProfile,
+        onRefreshFirebaseProfile: this._reloadFirebaseProfile,
+        onSendEmailVerification: this._doSendEmailVerification,
         getAuthorizationHeader: () =>
           this.props.authentication.getAuthorizationHeader(),
       },
     });
   }
 
-  _fetchUserProfile = () => {
+  _reloadFirebaseProfile = (callback?: FirebaseUser => void) => {
     const { authentication } = this.props;
 
     authentication.getFirebaseUser((err, firebaseUser: ?FirebaseUser) => {
       if (err && err.unauthenticated) {
         return this.setState(({ authenticatedUser }) => ({
           authenticatedUser: {
-            ...authenticatedUser,
+            ...this.state.authenticatedUser,
```

Comment:
Careful here, this could be a regression.

Reply:
interesting, must have come out of a bad conflict resolution. :+1: